ansible/roles/test/files/saitests/sai_qos_tests.py

```diff
@@ -528,17 +528,7 @@ def runTest(self):
         # or the leak out is simply less than expected as we have occasionally observed
         margin = 2
 
-        if asic_type == 'mellanox':
```

Comment:
the same optimization can also be done in lines 83, 670, 799, 1015, and many others...

Reply:
@wendani, agree with adriy, can you optimize that?

Reply:
https://github.com/Azure/sonic-mgmt/pull/1168